pkg/kubeadm/util/tokens.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package kubeadmutil
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	kubeadmapi "k8s.io/kubernetes/pkg/kubeadm/api"
+	utilrand "k8s.io/kubernetes/pkg/util/rand"
+)
+
+const (
+	TokenIDLen = 6
```

Comment:
> Is there a reason not to allow other sizes?

No, happy to make this any length.

> I could see us warning if these are under a certain size...

Perhaps.

Also we still need to write down the exect requirements for the length of token itself. As we said on the call, @dgoodwin and I got to assume that it has to be hex-encoded string representing 64 bits...